deploy.sh

```diff
@@ -14,7 +14,7 @@ GITPATH="$CURRENTDIR/" # this file should be in the base of your git repository
 # svn config
 SVNPATH="/tmp/$PLUGINSLUG" # path to a temp SVN repo. No trailing slash required and don't add trunk.
 SVNURL="http://plugins.svn.wordpress.org/$PLUGINSLUG/" # Remote SVN repo on wordpress.org, with no trailing slash
-SVNUSER="emspay" # your svn username
+SVNUSER="dave.ligthart" # your svn username
```

Comment:
@dmikolenko this is not necessary, the deployment ticket was for another project - please keep emspay as the deployment user.